specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databaseVulnerabilityAssessmentScans.json

```diff
@@ -0,0 +1,646 @@
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets the list of a database vulnerability assessment scan records": {
+            "$ref": "./examples/DatabaseVulnerabilityAssessmentScanRecordsListByDatabase.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/vulnerabilityAssessments/{vulnerabilityAssessmentName}/scans/{scanId}/export": {
+      "post": {
+        "tags": [
+          "DatabaseVulnerabilityAssessmentScansExport"
+        ],
+        "description": "Convert an existing scan result to a human readable format. If already exists nothing happens",
+        "operationId": "DatabaseVulnerabilityAssessmentScans_Export",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "databaseName",
+            "in": "path",
+            "description": "The name of the scanned database.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "vulnerabilityAssessmentName",
+            "in": "path",
+            "description": "The name of the vulnerability assessment.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "x-ms-enum": {
+              "name": "VulnerabilityAssessmentName",
+              "modelAsString": true
+            }
+          },
+          {
+            "name": "scanId",
+            "in": "path",
+            "description": "The vulnerability assessment scan Id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Scan result was converted successfully.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseVulnerabilityAssessmentScansExport"
```

Comment:
If scan result was already converted, does this POST call return that existing result?

Reply:
yes